tests/Hosting.Services.Web.UnitTests/Middlewares/UserIdentity/EmailBasedUserIdentityProviderTests.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Omex.Extensions.Hosting.Services.Web.Middlewares;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services.Web.UnitTests
+{
+	[TestClass]
+	public class EmailBasedUserIdentityProviderTests
+	{
+		[TestMethod]
+		public async Task TryWriteBytes_ReturnFalseIfNoEmail()
+		{
+			IUserIdentityProvider provider = new EmailBasedUserIdentityProvider();
+			(HttpContext context, _) = HttpContextHelper.CreateHttpContext();
+			Memory<byte> memory = new byte[provider.MaxBytesInIdentity];
+			(bool result, int bytesWritten) = await provider.TryWriteBytesAsync(context, memory).ConfigureAwait(false);
```

Comment:
You can remove the `.ConfigureAwait(false)`. This was only beneficial for old versions of .NET. It doesn't add anything now, and just makes the code more complex to read, so it's better to remove throughout